torch/onnx/symbolic_opset9.py

```diff
@@ -2249,7 +2249,23 @@ def _get_arange_dtype(dtype):
             dtype = 4  # default to int64
         return dtype
 
-    if len(args) == 5:
+    if len(args) == 2:
+        # aten::arange(Scalar end, Tensor out)
+        end = g.op("Unsqueeze", args[0], axes_i=[0])
+        dtype = 4  # default to int64
+        arange_tensor = g.op("Squeeze", nonzero(g, ones(g, end, dtype, None, None)), axes_i=[1])
+        return g.op("Cast", arange_tensor, to_i=sym_help.scalar_type_to_onnx[dtype])
+    elif len(args) == 4:
```

Comment:
Same comments as in opset 11.

Reply:
Only one/three lines are duplicated in opset 9, would it still be necessary to refactor?